.github/workflows/fedora.yml

```diff
@@ -92,11 +117,6 @@ jobs:
       - name: Install build dependencies
         run: bash ${OPENVINO_REPO}/install_build_dependencies.sh
```

Comment:
why we do not install them in the docker images?

Reply:
They are installed there, forgot to remove it from the workflow file; removed.